### PR TITLE
refactor: move plugin specific files to plugin directory

### LINKED ISSUE #103: Avoid plugin directories in the skin directory
As discussed in #102 and more specifically [here](https://github.com/liferay/liferay-ckeditor/pull/102#discussion_r465605154), we should avoid putting plugin specific code in the `skins` directory, and place it under the `plugins` directory.

### PR DESCRIPTION
**Pull request by julien**
Closes #103

Just in case, it might be worth waiting until #102 is merged to make sure there are no conflicts.

### CODE REVIEW

File: ck.sh [hunk 1]
```diff
@@ -56,15 +56,10 @@ case "$COMMAND" in
...
-				pluginsWithSkins=$(find ckeditor-dev/plugins -maxdepth 2 -mindepth 2 -type d -name skins)
-				for plugin in $pluginsWithSkins; do
-					pluginDir=$(dirname "$plugin")
-					pluginName=$(basename "$pluginDir")
-					pluginSkinPath="skins/moono-lexicon/plugins/$pluginName/skins/moono-lexicon"
-					if [[ -d "$pluginSkinPath" ]]; then
-						cp -r "$pluginSkinPath" "$pluginDir/skins"
-					fi
+				# Copy plugins
+				plugins=$(find plugins -maxdepth 1 -mindepth 1 -type d)
```
Comment (carloslancha) [inline ck.sh:60]: Can we just copy plugins folder here?

`cp -r "plugins" "ckeditor-dev"`

Comment (julien) [reply ck.sh]: :+1
sure let's keep it simple

**Comment by wincent** [comment]
FWIW, I don't think these are "chore:" commits; they are "refactor:".

**Comment by carloslancha** [comment]
LGTM 😄

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Leaving this one on hold due to https://github.com/liferay/liferay-ckeditor/pull/112

**Comment by carloslancha** [comment]
Continues in https://github.com/liferay/liferay-ckeditor/pull/128

**Pull request closed without merging**